modules/manage/pages/iceberg/choose-iceberg-mode.adoc

```diff
@@ -208,19 +215,22 @@ Protobuf::
 | int64 | long
 | sint64 | long
 | sfixed32 | int
-| sfixed64 | int
+| sfixed64 | long
 | string | string
 | bytes | binary
 | map | map
 |===
 
+There are some cases where the Protobuf type does not map directly to an Iceberg type and Redpanda applies the following transformations:
```

Comment:
@rockwotj @mattschumpert We received a [docs issue](https://redpandadata.atlassian.net/browse/DOC-1178) that points out some confusion with this section of the page. Are these changes on the right track to resolve the issue or do I need to make other changes?

Reply:
To the above table add message -> struct is the feedback that the docs ticket wanted